deploy/README.md

```diff
@@ -389,7 +389,7 @@ To run the pipeline, open the Synapse Studio for the Synapse workspace that you
     > SELECT *, ST_Distance(ST_SetSRID(ST_MakePoint(given_long, given_lat), 4326), location) AS dist FROM aioutputmodelschema.cvmodel
   - More documentation on queries can be found [here](https://postgis.net/workshops/postgis-intro/geography.html)
 
-# Running the pipeline (Custom Vision Model V2)
+# Running the pipeline (Custom Vision Model)
```

Comment:
The V2 section has been renamed. Was the older section removed?

Reply:
Removed, thanks for catching!